_posts/2015-03-27-ApiBlueprint-button.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Sharing is caring - socialize you're public API blueprint
+excerpt: Learn the art of API blueprint through socialization and collaboration.
+layout: post
+date: 2015-03-27 00:00:00 +0100
+author: vincenzo
+published: true
+commets: true
+---
+
+We have recently received a lot of feedback from users that would like to learn more about API Blueprint and how to present stunning and beautiful documentation.
```

Comment:
You're right, present is not the right verb. I would have use **show**.
The concept here is that a lot of people sees very well written documentation, but do not know how to replicate that.

So, given the new default blueprint **AND** the ability to view public blueprint...it should be a bit easier.

How would you modulate this concept?